roles/dns/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Install prerequisites
+  package:
+    name: python-dns
```

Comment:
Might be a good time to gather_facts about localhost and then start adding in blocks